src/joystream-lib/types.ts

```diff
@@ -29,6 +29,10 @@ export type VideoAssets = {
   video?: NewAsset
 }
 
+export type ContentIdArgs = {
+  [fieldName: string]: string
+}
```

Comment:
We should:
1. Update the name since this is no longer args as I understand. Maybe `ExtrinsicContentIds`? The name will be a better fit after Giza but that should come soon
2. Introduce some typesafety, currently this is prone to typos. This could be:
```tsx

type ExtrinsicChannelContentIds = {
  avatarPhotoId?: string
  coverPhotoId?: string
}

type ExtrinsicVideoContentIds = {
  thumbnailPhotoId?: string
  mediaId?: string
}

export type ExtrinsicContentIds = ExtrinsicChannelContentIds | ExtrinsicVideoContentIds
```